tests/system/providers/google/cloud/dataplex/example_dataplex.py

```diff
@@ -41,7 +41,7 @@
 from airflow.utils.trigger_rule import TriggerRule
 
 ENV_ID = os.environ.get("SYSTEM_TESTS_ENV_ID")
-PROJECT_ID = os.environ.get("SYSTEM_TESTS_GCP_PROJECT")
+PROJECT_ID = os.environ.get("SYSTEM_TESTS_GCP_PROJECT") or "project_id"
```

Comment:
```suggestion
PROJECT_ID = os.environ.get("SYSTEM_TESTS_GCP_PROJECT", "project_id")
```

should we make it this way for consistency?

Reply:
Ah yeah. one place left